torch/linalg/__init__.py

```diff
@@ -431,6 +431,45 @@
             [1, 2, 2, 2]])
 """)
 
+multi_dot = _add_docstr(_linalg.linalg_multi_dot, r"""
+linalg.multi_dot(tensors, *, out=None)
+
+Efficiently multiplies two or more matrices given by :attr:`tensors` by
+selecting the optimal matrix chain multiplication order.
+
+Every matrix in :attr:`tensors` must be 2D except for the first and last ones
```

Comment:
"Every matrix in ... 2D except for the first and last ones which can also be 1D" -> "Every tensor in ... 2D, except for the first and last which may be 1D."